documentation/subpacks.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: Subpacks
+parent: Documentation
+---
+
+# Subpacks
+
+## What are subpacks?
+
+Subpacks are what cause the gear icon to appear on packs.
```

Comment:
You seem to be using hard-returns a lot. I think its better to leave blocks like this on a single line (does your text editor wrap?)

This will allow the page to dynamically scale better.

Reply:
I'll fix that and yes my text editor does wrap automatically, I'll keep this in mind for future articles 😄

Reply:
Done :)